tests/Shipment/ShipmentTest.php

```diff
@@ -222,6 +222,15 @@ public function test_it_should_transform_shipment_to_json_api_with_all_inputs():
             ],
         ]);
 
+        $taxNumberMock = Mockery::mock(TaxIdentificationNumbers::class, [
```

Comment:
Variable says `$taxNumberMock` but it is actually a mock for a collection of tax identification numbers.
```suggestion
        $taxNumbersMock = Mockery::mock(TaxIdentificationNumbers::class, [
```

Reply:
Will rename this into taxNumberCollection since it then correlates to the entity itself.